fastapi_crudrouter/core/_base.py

```diff
@@ -90,6 +92,7 @@ def __init__(
                 response_model=self.schema,
                 summary="Get One",
                 dependencies=get_one_route,
+                responses=RESPONSES,
```

Comment:
```suggestion
                responses=[NOT_FOUND],
```
Minor: It may be cleaner to add the field `responses: [HTTPException] = None` to the `_add_api_route`. In the `_add_api_route` function. You could then extract the status code / detail of the exception, reducing code duplication.

Reply:
This won't work unfortunately - the responses must be a mapping, so we would need something like a dictionary merge. But I guess we could also just define a `RESPONSES` to contain everything needed.

What would you prefer? 🙂 

The type needs to be `Optional[Dict[Union[int, str], Dict[str, Any]]]` and it fails with `TypeError: 'list' object is not a mapping` if you try to pass a list.

Reply:
I was imagining either

```python
NOT_FOUND_RESPONSE = {"404": {"detail": "Item not found"}}
FORBIDDEN_RESPONSE = {"403": {"detail": "Forbidden"}}
INTERNAL_SERVER_ERROR_RESPONSE = {"500": {"detail": "Server error"}}
RESPONSES = NOT_FOUND_RESPONSE | FORBIDDEN_RESPONSE | INTERNAL_SERVER_ERROR_RESPONSE
```

or just

```python
RESPONSES = {
    "404": {"detail": "Item not found"}, 
    "403": {"detail": "Forbidden"}, 
    "500": {"detail": "Server error"}
}
```

Reply:
I implemented the last one in eb489bb, but let me know if you prefer something else 👏